documentation/docs/steps/mtaBuild.md

```diff
@@ -1,44 +1,10 @@
-# mtaBuild
+# ${docGenStepName}
 
-## Description
+## ${docGenDescription}
 
-Executes the SAP Multitarget Application Archive Builder to create an mtar archive of the application.
+## ${docGenParameters}
 
-Before doing this, validates that SAP Multitarget Application Archive Builder exists and the version is compatible.
```

Comment:
I think we should add this as a prerequisite to the docs if a custom image is used.

Reply:
But the library is not validating the tool anymore. It is in the prerequisites, then the user should take care of this.

Reply:
Ah no, it is not in the prerequisites, I will add it as prerequisite then. What do you think?

Reply:
Yes, I think it makes sense to mention it here.